src/System.Net.Security/src/System/Net/SecureProtocols/NegoState.Unix.cs

```diff
@@ -19,140 +19,135 @@ namespace System.Net.Security
     //
     // This is part of the NegotiateStream PAL.
     //
-    internal class NegoState
+    internal partial class NegoState
```

Comment:
Why are we postponing so much stuff to an "Address PR feedback" issue?  Most of these "will add it there" issues could be addressed in the code more quickly than it would take to summarize the relevant details in the follow-up issue.